php.ini-development

```diff
@@ -1374,16 +1416,14 @@ session.gc_probability = 1
 ; the gc will run on any give request. Increasing this value to 1000 will give you
 ; a 0.1% chance the gc will run on any give request. For high volume production servers,
 ; this is a more efficient approach.
-; Default Value: 100
-; Development Value: 1000
-; Production Value: 1000
```

Comment:
The same as above. I intended to remove probability based GC at first, but I decided not to. It's leftover and will be fixed.